lib/ui/platform_dispatcher.dart

```diff
@@ -69,6 +69,19 @@ const double _kUnsetGestureSetting = -1.0;
 // See embedder.cc::kFlutterKeyDataChannel for more information.
 const String _kFlutterKeyDataChannel = 'flutter/keydata';
 
+class RootIsolateToken {
+  final int _rootIsolateId;
+  const RootIsolateToken._(this._rootIsolateId);
```

Comment:
trivial nit: constructor first per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#order-other-class-members-in-a-way-that-makes-sense

Reply:
done